Lib/asyncio/tools.py

```diff
@@ -232,20 +234,56 @@ def _get_awaited_by_tasks(pid: int) -> list:
         sys.exit(1)
 
 
-def display_awaited_by_tasks_table(pid: int) -> None:
+class TaskTableOutputFormat(Enum):
+    table = "table"
+    csv = "csv"
+    bsv = "bsv"
+    # As per the words of the asyncio 🍌SV spec lead:
+    # > 🍌SV is not just a format. It’s a lifestyle. A philosophy.
+    # https://www.youtube.com/watch?v=RrsVi1P6n0w
+
+
+_header = ('tid', 'task id', 'task name', 'coroutine stack', 'awaiter chain', 'awaiter name', 'awaiter id')
+
+
+def display_awaited_by_tasks_table(
+        pid: int,
+        format_: TaskTableOutputFormat | str = TaskTableOutputFormat.table
+    ) -> None:
```

Comment:
Also remove the underscore from `format_`?
```suggestion
def display_awaited_by_tasks_table(pid, format_=TaskTableOutputFormat.table):
```

Reply:
I have not removed the type annotations because they were already otherwise present in this file

Reply:
They're needlessly verbose here, hence the suggestion.